.github/workflows/ci.yml

```
@@ -142,16 +142,9 @@ jobs:
         id: electron-unit-tests
         run: DISPLAY=:10 ./scripts/test.sh --runGlob "**/sql/**/*.test.js" --coverage
 
-      - name: Run Extension Unit Tests (Continue on Error) # {{SQL CARBON EDIT}} Run extension unit tests (continue on error)
-        id: electron-extension-unit-tests-continue-on-error
+      - name: Run Extension Unit Tests # {{SQL CARBON EDIT}} Rename to core for clarity
```

Missed this in the original PR that changed these names

```suggestion
      - name: Run Extension Unit Tests # {{SQL CARBON EDIT}} Rename to extension for clarity
```